tools/prereqs/setup_terraform.sh

```diff
@@ -40,6 +40,7 @@ fi
 TERRAFORM_VERSION=`curl -L -s https://github.com/hashicorp/terraform/releases/latest | grep archive | grep zip | awk -F"/v" '{print $2}' | awk -F".zip" '{print $1}'`
 curl -LO -s https://releases.hashicorp.com/terraform/$TERRAFORM_VERSION/terraform_"$TERRAFORM_VERSION"_$arch.zip
 
+sudo apt-get install unzip
```

Comment:
this will fail on macos as the package manager is homebrew, not apt

Reply:
Will update